spinalcordtoolbox/deepseg/models.py

```diff
@@ -201,6 +202,16 @@
         "thr": None,  # Images are already binarized
         "default": False,
     },
+    "totalspineseg": {
+         # NB: Rather than hardcoding the URLs ourselves, use the URLs from the totalspineseg package.
+         # This means that when the totalspineseg package is updated, the URLs will be too, thus triggering
+         # a re-installation of the model URLs
+         "url": tss_init.ZIP_URLS,
```

Comment:
There is a new problem with the import time.

Because we need to access `init_inference` in order to get the `ZIP_URLS` dictionary, we also have to import the heavy `torchio` package at the same time.

@NathanMolinier would you be able to place the `ZIP_URLS` object in a location where there are no other unnecessary imports? (That way, accessing `ZIP_URLS` won't incur a big slowdown.)

Reply:
Yes I can try that

Reply:
I moved `ZIP_URLS` to `totalspineseg.utils.utils`

Reply:
I'm very sorry, but because `utils.py` is a subset of `utils`, and `utils` contains `augment`, the slowdown is the exact same as before. :sweat: 

![image](https://github.com/user-attachments/assets/7ccabe37-e104-4bc1-b233-605b715351b4)

Reply:
I am not sure what the solution is, actually... the import graph above shows `totalspineseg.utils` -> `totalspineseg` -> `totalspineseg.utils.augment`.

Let me think about how to best approach this. Give me one moment.

Reply:
Turns out that SCT can fetch the URLs directly from totalspineseg: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4690/commits/56f2ae4906c5959df39ec9f3e9eeb3a5c82ed0b4